React/Views/RCTView.m

```diff
@@ -184,6 +184,53 @@ - (BOOL)didActivateAccessibilityCustomAction:(UIAccessibilityCustomAction *)acti
   return YES;
 }
 
+- (NSString *)accessibilityValue
+{
+  NSMutableString *value = [NSMutableString stringWithString:@""];
```

Comment:
Rather than mutate a `NSMutableString`, build up a `NSMutableArray` and use `componentsJoinedByString:` at the end.